1085 finalize button styles

minump opened the pull request:
## Description
Button styles changed to match zeplin design.
Used button tag with onclick function instead of anchor "a" tags.
Updated "Contribute +" button in topnav. Updated login/logout buttons. Updated buttons in contribution_details page.

**Fixes #1085**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [ ] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [ ] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [ ] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

sandeep-ps commented:
It's good that you could change the anchor elements back to the button elements. Thanks.

sandeep-ps commented:
The focus states of Contribute+ and Save buttons look slightly off. The rest looks good to me. We might be waiting for Contribute+ icon for the focus style, but if you could update just the border, I think it'll be good. Thanks.
<img width="140" alt="image" src="https://user-images.githubusercontent.com/4306604/223759968-52f4a2d4-17eb-45fd-bb40-4158dcc2f693.png">
<img width="92" alt="image" src="https://user-images.githubusercontent.com/4306604/223759863-af2316f5-e879-4332-bb3b-369950f0253d.png">

minump commented:
> 

Updated focus styles.

Ready for review @sandeep-ps

sandeep-ps commented:
Thanks for fixing the previous items. I discovered a couple more things when re-testing this:
1. Focus styles of these buttons need to be updated. On click, the styles don't match the design.
<img width="418" alt="image" src="https://user-images.githubusercontent.com/4306604/224123919-1f6aff95-7ad1-4aa9-86e3-a6542492b82f.png">
2. The dual line still appears when clicking on the login, logout, and contribute buttons. You can see this by clicking the button and holding onto it. Please update this.

I couldn't find any other issues. Thanks.

minump commented:
> Thanks for fixing the previous items. I discovered a couple more things when re-testing this:
> 
> 1. Focus styles of these buttons need to be updated. On click, the styles don't match the design.
> 
> <img alt="image" width="418" src="https://user-images.githubusercontent.com/4306604/224123919-1f6aff95-7ad1-4aa9-86e3-a6542492b82f.png">
> 
> 2. The dual line still appears when clicking on the login, logout, and contribute buttons. You can see this by clicking the button and holding onto it. Please update this.
> I couldn't find any other issues. Thanks.

1. Updated buttons styles for "Show All" "Capabilities" and "Talents"
2. Not able to find a solution for this. Tried multiple stylings such as 
```
outline:none;
-moz-outline-style: none;
   -webkit-appearance: none;
   -moz-box-shadow: none;
   -webkit-box-shadow: none;
box-shadow:none;
```
None seems to work.

minump commented:
> > Thanks for fixing the previous items. I discovered a couple more things when re-testing this:
> > 
> > 1. Focus styles of these buttons need to be updated. On click, the styles don't match the design.
> > 
> > <img alt="image" width="418" src="https://user-images.githubusercontent.com/4306604/224123919-1f6aff95-7ad1-4aa9-86e3-a6542492b82f.png">
> > 
> > 2. The dual line still appears when clicking on the login, logout, and contribute buttons. You can see this by clicking the button and holding onto it. Please update this.
> >    I couldn't find any other issues. Thanks.
> 
> 1. Updated buttons styles for "Show All" "Capabilities" and "Talents"
> 2. Not able to find a solution for this. Tried multiple stylings such as
> 
> ```
> outline:none;
> -moz-outline-style: none;
>    -webkit-appearance: none;
>    -moz-box-shadow: none;
>    -webkit-box-shadow: none;
> box-shadow:none;
> ```
> 
> None seems to work.

2. Inline style worked. Added inline "style=outline:none" for the three buttons.

minump commented:
Updated code according to feedback. Ready for review. @sandeep-ps

sandeep-ps reviewed (approved):
The changes look good to me. Approving. Thanks.

minump commented:
merging to develop
